core/kazoo_endpoint/src/kz_endpoint.erl

```diff
@@ -1552,6 +1553,13 @@ maybe_set_call_forward({Endpoint, Call, CallFwd, CCVs}) ->
                        )
     }.
 
+-spec maybe_rtcp_mux(ccv_acc()) -> ccv_acc().
+maybe_rtcp_mux({Endpoint, Call, CallFwd, CCVs} = Acc) ->
+    case kz_json:get_boolean_value([<<"media">>, <<"rtcp_mux">>], Endpoint, 'undefined') of
```

Comment:
@sergey-safarov last one, can you change this call from `kz_json:get_boolean_value/3` to `kz_json:get_boolean_value/2` ?